tox.ini

```diff
@@ -35,9 +35,9 @@ setenv =
 # https://github.com/shazow/urllib3/pull/930
 deps =
     py{26,27}-oldest: cffi<=1.7
-    py{26,27}-oldest: cryptography==0.8
+    py{26,27}-oldest: cryptography==1.8.1
```

Comment:
Is there a reason these deps were older? They're already not equivalent to what we're pinning in certbot-auto (1.5.3, 16.2.0), which is what I would have expected them to be.